docs/csharp/language-reference/operators/stackalloc.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "stackalloc operator - C# Reference"
+ms.custom: seodec18
+ms.date: 06/10/2019
+f1_keywords: 
+  - "stackalloc_CSharpKeyword"
+helpviewer_keywords: 
+  - "stackalloc operator [C#]"
+---
+# stackalloc operator (C# Reference)
+
+The `stackalloc` operator allocates a block of memory on the stack. A stack allocated memory block created during the method execution is automatically discarded when that method returns. You cannot explicitly free memory allocated with the `stackalloc` operator. A stack allocated memory block is not subject to [garbage collection](../../../standard/garbage-collection/index.md) and doesn't have to be pinned with the [`fixed` statement](../keywords/fixed-statement.md).
+
+You can assign the result of the `stackalloc` operator to a variable of one of the following types:
+
+- A [pointer type](../../programming-guide/unsafe-code-pointers/pointer-types.md), as the following example shows:
```

Comment:
I think we should reverse the order of these types: Because `Span<T>` can be used in a safe context, we should encourage it.

Reply:
@BillWagner I agree with you: swapped the types. Thanks!